docs/user/openstack/README.md

```diff
@@ -202,6 +207,24 @@ clouds:
 The file can contain information about several clouds. For instance, the example above describes two clouds: `shiftstack` and `dev-evn`.
 In order to determine which cloud to use, the user can either specify it in the `install-config.yaml` file under `platform.openstack.cloud` or with `OS_CLOUD` environment variable. If both are omitted, then the cloud name defaults to `openstack`.
 
+### Self Signed OpenStack CA certificates
+
+If your OpenStack cluster uses self signed CA certificates for endpoint auth, you will need a few additional steps to run the installer. First, make sure that the host running the installer trusts your CA certificates.
+
+```sh
+sudo cp ca.crt.pem /etc/pki/ca-trust/source/anchors/
+sudo update-ca-trust extract
+```
+
+Next, you should add the `cacert` key to your `clouds.yaml`. Its value should be a valid path to your CA cert that does not require root privelage.
+
+```yaml
+clouds:
+  shiftstack:
+    auth: ...
+    cacert: "ca.crt.pem"
+```
+
```

Comment:
Can you add a note saying:

> Note that in deployments configured with Kuryr, the provided CA certificate will completely replace default CAs used by Kuryr services. This means it should be valid for all the OpenStack endpoints of underlying cloud.

I'm not 100% sure this is really needed if we're now specifying this in `cacert` option of the cloud. It seems implicit that the CA should be valid for all cloud.

Reply:
I don't think we need this note, as you say the cacert referenced from the clouds.yaml should cover all of the OpenStack endpoints.

Reply:
I just followed the rule of thumb that more docs are normally better. And Kuryr's behavior here is different than of the rest components that append the provided certs to system list.

Reply:
It could be a problem in environments with egress proxy. 
https://github.com/openshift/enhancements/blob/master/enhancements/proxy/global-cluster-egress-proxy.md

Isn't it valid for kuryr pod?

Reply:
Right, it only replaces the CA cert for connections to underlying OpenStack cloud. I assume that having that cloud behind egress proxy would make no sense.

Am I missing anything here?

Reply:
Yeah, you're right I think. I also don't see it being a valid scenario.